.ci/script_cmake.sh

```diff
@@ -8,4 +8,4 @@ mkdir build
 cd build
 cmake -DCMAKE_BUILD_TYPE=${BUILD_TYPE} -DTREAT_WARNINGS_AS_ERRORS=ON ..
 make -j4 tests
-make test
+ctest
```

Comment:
Why?

Reply:
Just to see if it makes any difference that turned out not. Feel free to revert it.